devel/libcoverart/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        metabrainz libcoverart 1.0.0 release-
+github.tarball_from releases
+categories          devel
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-2.1+
+
+description         C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+long_description    libcoverart is a C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+homepage            https://musicbrainz.org/doc/libcoverart
+
+checksums           rmd160  2c3d15b3399b16b842cde85fa02401caf72e4cdb \
+                    sha256  5d9d4a4c46f35754e016affab83957961a364d7a4c7d64447f73bff22a300f44 \
+                    size    57933
+
+depends_lib         \
+                    port:neon \
+                    port:jansson \
+                    port:libxml2
+
+depends_build       \
+                    port:pkgconfig
+
+PortGroup           cmake 1.0
+
+build.target
+
+livecheck.type      regex
+livecheck.url       [lindex ${master_sites} 0]
+livecheck.regex     "${name}-(\\d+(?:\\.\\d+)*)${extract.suffix}"
```

Comment:
This doesn't work:

```
$ port livecheck
Error: cannot check if libcoverart was updated (The requested URL returned error: 404 Not Found)
```

You should remove these three livecheck lines.

Usually the github portgroup's default livecheck should be fine.

In this case, since they offer beta versions and we don't want to find those, use:

```
github.livecheck.regex {([0-9.]+)}
```